lib/cpp/test/TransportTest.cpp

```diff
@@ -1016,7 +1019,7 @@ struct global_fixture {
   }
 };
 
-BOOST_GLOBAL_FIXTURE(global_fixture)
+BOOST_GLOBAL_FIXTURE(global_fixture);
```

Comment:
boost-1.59 rewrote boost::unit_test and they left the semicolon to the test implementer.  With clang-3.8 it won't compile without this.